README.md

```diff
@@ -139,10 +139,3 @@ Give a ⭐️ if you like this project!
 ## 📝 License
 
```

Comment:
- [ ] Some sections in your readme file contains information about math magician that is not related to this project. Kindly fix these errors.

  ![uiff](https://user-images.githubusercontent.com/81468675/207298746-58cdc116-58e7-4eef-8483-5a209ff12ac8.jpg)

Reply:
Ohh yeah okay, thank you for your feedback🙏